experiments/ClimaEarth/components/atmosphere/climaatmos.jl

```diff
@@ -60,19 +60,15 @@ function ClimaAtmosSimulation(atmos_config)
 
     if hasmoisture(integrator)
         ρ_flux_q_tot = integrator.p.precomputed.sfc_conditions.ρ_flux_q_tot
-        surface_rain_flux = integrator.p.precipitation.surface_rain_flux
-        surface_snow_flux = integrator.p.precipitation.surface_snow_flux
+        surface_rain_flux = integrator.p.precomputed.surface_rain_flux
+        surface_snow_flux = integrator.p.precomputed.surface_snow_flux
```

Comment:
Could you change this in such a way that we can run with both the new and old version of ClimaAtmos? Something like
```suggestion
if pkgversion(CA) >= v"0.27.6"
        surface_rain_flux = integrator.p.precomputed.surface_rain_flux
        surface_snow_flux = integrator.p.precomputed.surface_snow_flux
 else
         surface_rain_flux = integrator.p.precipitation.surface_rain_flux
        surface_snow_flux = integrator.p.precipitation.surface_snow_flux
 end
```